community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tests/test_util.py

```diff
@@ -199,16 +200,63 @@ def test_nodeset_reservation_err(nodeset, err):
 def test_nodeset_reservation_ok(nodeset, policies, expected):
     lkp = util.Lookup(TstCfg())
     lkp._get_reservation = Mock()
-    
+
     if not expected:
         assert lkp.nodeset_reservation(nodeset) is None
         lkp._get_reservation.assert_not_called()
         return
-    
+
     lkp._get_reservation.return_value = {
         "resourcePolicies": {i: p for i, p in enumerate(policies)},
     }
     assert lkp.nodeset_reservation(nodeset) == expected
     lkp._get_reservation.assert_called_once_with(expected.project, expected.zone, expected.name)
-    
-    
+
+
+@pytest.mark.parametrize(
+    "job_info,expected_job",
+    [
+        (
+            """JobId=123
+            TimeLimit=02:00:00
+            JobName=myjob
+            JobState=PENDING
+            ReqNodeList=node-[1-10]""",
+            util.Job(
+                id=123,
+                duration=timedelta(days=0, hours=2, minutes=0, seconds=0),
+                name="myjob",
+                job_state="PENDING",
+                required_nodes="node-[1-10]"
```

Comment:
Please add, or update test that involves long `ReqNodeList`, e.g. `node-[1-10],grob-pop-[2,1,44-77]`